dynamic_bitset.html

```diff
@@ -570,12 +573,12 @@ <h3><a id="constructors">Constructors</a></h3>
 <hr />
 <pre>
 <a id=
-"cons1">dynamic_bitset</a>(const Allocator&amp; alloc = Allocator())
+"cons0">dynamic_bitset</a>()
 </pre>
 
-<b>Effects:</b> Constructs a bitset of size zero. A copy of the
-<tt>alloc</tt> object will be used in subsequent bitset
-operations such as <tt>resize</tt> to allocate memory.<br />
+<b>Effects:</b> Constructs a bitset of size zero. The allocator
+for this bitset is a default-constructed object of type
+<tt>Allocator</tt>.<br />
  <b>Postconditions:</b> <tt>this-&gt;size() == 0</tt>.<br />
  <b>Throws:</b> An allocation error if memory is exhausted
```

Comment:
This `Throws:` isn't right. When default-constructing a `dynamic_bitset` there won't be an "allocation error if memory is exhausted". What could throw is if the `Allocator` constructor throws an exception. i.e. We default construct a `vector<T, Allocator>` and the `vector()` constructor is `noexcept(noexcept(Allocator()))`. i.e. In this constructor we aren't allocating anything.